### PR TITLE
Update Application Connector documentation

### LINKED ISSUE #9316: Application Connector tutorial unable to hit the events endpoint
**Description**
One of the steps of the [Application Connector tutorial](https://kyma-project.io/docs/components/application-connector/#tutorials-tutorials) is to call the Application Registry and Event Service on a local deployment using crt and key. I was able to use the cert/key to call the `v1/metadata/services` endpoint but was not able to trigger the `v1/events` endpoint.

Specifically the following code throws `{"code":405,"error":"Method not allowed."}` error. 
```
curl https://gateway.kyma.local/{APP_NAME}/v1/events --cert {CERT_FILE_NAME}.crt --key {KEY_FILE_NAME}.key -k
```

This also breaks the [Trigger a Function with events](https://kyma-project.io/docs/components/application-connector/#tutorials-trigger-a-function-with-events) as it contains steps to send an event by curling the `v1/events` endpoint: `https://gateway.{CLUSTER_DOMAIN}/$APP_NAME/v1/events -k --cert {CERT_FILE_NAME}.crt --key {KEY_FILE_NAME}.key -d`

**Expected result**
Status 2xx

**Actual result**
```
{"code":405,"error":"Method not allowed."}
```

**Steps to reproduce**
Follow the steps of [Application Connector Tutorial](https://kyma-project.io/docs/components/application-connector/#tutorials-tutorials) until the [Call the Application Registry and Event Service on local deployment](https://kyma-project.io/docs/components/application-connector/#tutorials-get-the-client-certificate-call-the-application-registry-and-event-service-on-local-deployment)

### PR DESCRIPTION
**Pull request by majakurcius**
**Description**

Changes proposed in this pull request:

- Fix the name of the value to change in the Istio Secret (`ca.crt` -> `cacert`) 
- Add a note that the user is responsible for renewing and rotating the client certificates for their integration
- Fix the curl command to call Event Service 
- Apply minor editorial fixes

**Related issues**
Resolves #9316

### CODE REVIEW

File: docs/application-connector/08-11-rotate-root-ca.md [hunk 1]
```diff
@@ -10,7 +10,9 @@ Two different components use the Root CA certificate. As a result, the certifica
   - The `connector-service-app-ca` Connector Service CA Secret responsible for signing certificate requests
   - The `app-connector-certs` Istio Secret responsible for security in the Connector Service API
 
-Keeping both Secrets up-to-date is vital for the security of your implementation as it guarantees that the Connector Service issues proper certificates and no unregistered Applications can access its API.
+Keeping both Secrets up-to-date is vital for the security of your implementation as it guarantees that the Connector Service issues proper certificates and no unregistered Applications can access its API. 
+
+> **NOTE:** You as the user are responsible for rotating and renewing the certificates.
```
Comment (akgalwas) [inline docs/application-connector/08-11-rotate-root-ca.md:15]:
```suggestion
> **NOTE:** You as the user are responsible for rotating and renewing the client certificates.
```

File: docs/application-connector/08-11-rotate-root-ca.md [hunk 2]
```diff
@@ -10,7 +10,9 @@ Two different components use the Root CA certificate. As a result, the certifica
   - The `connector-service-app-ca` Connector Service CA Secret responsible for signing certificate requests
```
Comment (akgalwas) [inline docs/application-connector/08-11-rotate-root-ca.md:10]: This is just a general note. Users need to bother with root CA creation only if they manage the cluster. In Compass scenario the root CA certificate is managed by the Compass Runtime Agent and no manual rotation is needed.

File: docs/application-connector/08-11-rotate-root-ca.md (hunk 1 above)
Comment (akgalwas) [inline docs/application-connector/08-11-rotate-root-ca.md:15]: We need to be cautious here, to not make user confused. The tutorial concerns root CA whereas the note concerns client certificates

**Review by akgalwas — DISMISSED** [review]

File: docs/application-connector/08-11-rotate-root-ca.md [hunk 3]
```diff
@@ -30,63 +30,63 @@ To successfully rotate a soon-to-expire CA certificate, replace it with a new ce
...
 
-1. Encode the newly created certificate with base64.
+3. Encode the newly created certificate with base64.
   
    ```bash
    cat new-ca.crt | base64
    ```
 
-1. Replace the old certificate in the Connector Service CA Secret. Edit the Secret and replace the `ca.crt` value with the new base64-encoded certificate.
+4. Replace the old certificate in the Connector Service CA Secret. Edit the Secret and replace the `ca.crt` value with the new base64-encoded certificate.
```
Comment (klaudiagrz) [inline docs/application-connector/08-11-rotate-root-ca.md:47]: Just to make sure - this `ca.crt` shouldn't be replaced with `cacert`, is that right?

File: docs/application-connector/08-09-renew-client-cert.md [hunk 4]
```diff
@@ -4,6 +4,8 @@ type: Tutorials
 ---
 
 By default, a client certificate you generate when you connect an external solution to Kyma is valid for 92 days. Follow this tutorial to renew a client certificate.
+ 
+> **NOTE:** You as the user are responsible for rotating and renewing the certificates.
```
Comment (klaudiagrz) [inline docs/application-connector/08-09-renew-client-cert.md:8]:
```suggestion
> **NOTE:** You, as the user, are responsible for rotating and renewing the certificates.
```

Comment (klaudiagrz) [reply docs/application-connector/08-09-renew-client-cert.md:8]: As discussed, it's a tempting change but I understand if you decide to skip the commas.

File: docs/application-connector/08-11-rotate-root-ca.md (hunk 3 above)
Comment (majakurcius) [reply docs/application-connector/08-11-rotate-root-ca.md:47]: It shouldn't. This is due to some changes introduced by another team and going with another name.

File: docs/application-connector/08-09-renew-client-cert.md (hunk 4 above)
Comment (majakurcius) [reply docs/application-connector/08-09-renew-client-cert.md:8]: It should emphasize that it's the USER's responsibility (with the stress on the user), so let's go without them :)

**Review by klaudiagrz — APPROVED** [review]

**Pull request merged**